modules/ROOT/pages/exportword.adoc

```diff
@@ -0,0 +1,65 @@
+= Export Word
+:navtitle: exportword
+:description: The export to Word feature lets you generate a .docx file directly from the editor.
+:description_short: Generate a .docx file directly from the editor.
+:keywords: plugin, exportword.
+:pluginname: Export to Word
+// values for <tier>
+// opensource plugin = tieroneplan
+// premium plugin    = tiertwoplan or tierthreeplan or enterpriseplan
+:pluginminimumplan: <tier>
+:plugincode: exportword
+:plugincategory: premium
+
+include::partial$misc/admon-paid-addon-pricing.adoc[]
+
+include::partial$misc/admon-requires-7.0v.adoc[]
+
+The export to Microsoft Word feature collects the HTML generated with the `tinymce.editor.getContent()` method and the default editor content styles combined with the styles provided by you in the configuration. It then sends them to the {productname} Cloud Services HTML to DOCX converter service. The service generates a Word file and returns it to the user’s browser so they can save it in the Word format on their disk.
+
+[NOTE]
+The generated `.docx` file may not be fully compatible with older versions of Word.
+
+The complementary pagination feature allows you to see where page breaks would be after you export the document to Word. Thanks to the live preview, the user can fine-tune the structure of the output document when editing it. The pagination feature also shows you the page count and lets you navigate between the document pages.
```

Comment:
I don't know about any live preview for this feature. It seems this is a left over and should be removed ?

Reply:
```suggestion
```